### PR TITLE
feat: added card component to Component Library

### PR DESCRIPTION
**Pull request by dev-prabhat**

### CODE REVIEW

**Review by itsmebipul00 — APPROVED** [review]

**Comment by itsmebipul00** [comment]
Nice work!

**Review by archit7448 — APPROVED** [review]
looks good to merge

File: document/alert/alert.html [hunk 1]
```diff
@@ -41,6 +41,7 @@ <h1 class="head-lg margin-xs">Components</h1>
         <a class="d-block btn-link" href="/document/button/button.html"
           >Button</a
         >
+        <a class="d-block btn-link" href="/document/card/card.html">Card</a>
```
Comment (iRohitGaur) [inline document/alert/alert.html:44]: Suggestion: d-block class name is not explaining what it does. Try to use better names

**Review by iRohitGaur — APPROVED** [review]
Good to merge 👍🏻

File: document/alert/alert.html (hunk 1 above)
Comment (dev-prabhat) [reply document/alert/alert.html:44]: like d-flex == display : flex then d-block == display: block why not?

Comment (iRohitGaur) [reply document/alert/alert.html:44]: Makes sense now. Thanks for explaining.

**Pull request merged**